gfx/NeoDays/pngs_tiles_10x10/monsters/triffid/mon_creeper_vine.json

```diff
@@ -0,0 +1,33 @@
+{
+  "id": "mon_creeper_vine",
+  "fg": "mon_creeper_vine_0",
+  "rotates": false,
+  "//": "below doesnt work (as intended? it uses '_2')",
+  "multitile": true,
```

Comment:
I don't think monsters can use multitile like this

Reply:
I'll move them to scratch.